skyportal/models.py

```diff
@@ -239,9 +245,18 @@ class Photometry(Base):
     mjd = sa.Column(sa.Float)  # mjd date
     time_format = sa.Column(sa.String, default='iso')
     time_scale = sa.Column(sa.String, default='utc')
-    mag = sa.Column(sa.Float)
-    e_mag = sa.Column(sa.Float)
+
```

Comment:
I think the only things that need to be merged here are the change to this file, but I would also keep mag + e_mag as fields.

Reply:
ZTF alerts do not provide flux/fluxerr, but offline forced photometry does. Best to have all of these as fields, and require either mag or flux, but not both.

Danny